src/api/system.c

```diff
@@ -52,9 +52,10 @@ struct HitTestInfo {
 typedef struct HitTestInfo HitTestInfo;
 
 static HitTestInfo window_hit_info[1] = {{0, 0, 0}};
+static int window_hit_ref = LUA_NOREF;
 
-#define RESIZE_FROM_TOP 0
-#define RESIZE_FROM_RIGHT 0
+#define RESIZE_FROM_TOP 1
+#define RESIZE_FROM_RIGHT 1
```

Comment:
How does it behave with the scrollbar?

Reply:
This doesn't look good. The scrollbar is heavily offsetted.

https://user-images.githubusercontent.com/20792268/174535909-b8116ab3-39a9-4256-aae9-b37dca2304a9.mp4